interfaces/builtin/i2c.go

```diff
@@ -129,7 +128,10 @@ func (iface *i2cInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 	}
 
 	cleanedPath := filepath.Clean(path)
-	spec.AddSnippet(fmt.Sprintf(i2cConnectedPlugAppArmorPath, cleanedPath, strings.TrimPrefix(path, "/dev/")))
+	spec.AddSnippet(fmt.Sprintf(i2cConnectedPlugAppArmorPath, cleanedPath))
+	spec.AddParametricSnippet([]string{
+		"/sys/devices/platform/{*,**.i2c}/" /* ###PARAM### */, "/** rw,  # Add any condensed parametric rules",
+	}, strings.TrimPrefix(path, "/dev/"))
```

Comment:
Since i2cControlDeviceNodePattern enforces /dev/i2c-N, like with iio and spi, can this be updated to use:
```
spec.AddParametricSnippet([]string{
	"/sys/devices/platform/{*,**.i2c}/i2c-" /* ###PARAM### */, "/** rw,  # Add any condensed parametric rules",
}, strings.TrimPrefix(path, "/dev/i2c-"))
```

Reply:
Indeed, this is done now.